ktor-server/ktor-server-plugins/ktor-server-websockets/jvmAndNix/test/io/ktor/tests/websocket/RawWebSocketTest.kt

```diff
@@ -202,8 +197,9 @@ class RawWebSocketTest {
     }
 
     private fun runTest(block: suspend CoroutineScope.() -> Unit) {
-        runBlocking(CoroutineName("test-${test.methodName}")) {
+        runBlocking(CoroutineName("test-")) {
```

Comment:
Could we find a decent replacement?

Reply:
I think, that the only possible way is to use Throwable stacktrace or some compiler plugin.
For now I return it back via expect-actual